packages/react-scripts/template/src/Toast.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './Toast.css';
+
+export default class Toast extends React.Component {
+  state = {
+    active: true,
+  };
+  componentDidMount() {
+    setTimeout(
+      () => {
+        this.setState({
+          active: false,
+        });
+      },
+      this.props.timeout
+    );
+  }
+  render() {
+    return (
+      <div className={`Toast ${this.state.active ? 'active' : 'inactive'}`}>
+        {this.props.children}
+      </div>
+    );
+  }
+}
+
+Toast.defaultProps = {
+  timeout: 3000,
```

Comment:
A 3s timeout is consistent with some of the other implementations that have opted for this pattern in case anyone is wondering: https://github.com/GoogleChrome/voice-memos/blob/bcb225a41d3dfcec5e9805adf33d27995840c9f0/src/scripts/libs/Toaster.js#L41.